### DISCUSSION TITLE
osm2pgsql-replication: command not found on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by leongal** [osm2pgsql-dev/osm2pgsql#1695] [General]
Hi Guys
I installed osm2pgsql on my Ubuntu 20.04.
And when I try to use osm2pgsql-replication, it says osm2pgsql-replication: command not found.
I can see my osm2pgsql version is 1.4.1 and osm2pgsql-replication is available at  >=1.4.2.
How can I upgrade my osm2pgsql version?
Also if I am going to use https://raw.githubusercontent.com/openstreetmap/osm2pgsql/master/scripts/osm2pgsql-replication this script for osm2pgsql-replication, where should I copy it to use as a command?
Thanks in advance!

**Comment by SomeoneElseOSM** [comment]
Not really a bug, but https://switch2osm.org/serving-tiles/updating-as-people-edit-osm2pgsql-replication/ , which links to https://osm2pgsql.org/doc/install.html and https://osm2pgsql.org/doc/manual.html#updating-an-existing-database , might help you.  See also https://switch2osm.org/serving-tiles/updating-as-people-edit-pyosmium/ which is a slightly more flexible alternative.

**Comment by leongal** [comment]
![image](https://user-images.githubusercontent.com/105981020/180214036-5b05c74c-d893-4bd6-812c-29a934a58d06.png)
As you can see on Image, I did `sudo apt-get install osm2pgsql` and it installed osm2pgsql of version 1.4.1.
Do you have any opinion about this?
To update from `osm2pgsql 1.4.1`  to `osm2pgsql 1.4.2`, how can I do?

**Reply by SomeoneElseOSM** [reply]
You've installed the version that is packaged with your operating system.  If you want to run a later version, you can either upgrade from 20.04 to 22.04 ([here](https://www.reddit.com/r/Ubuntu/comments/vrfxt6/roughly_when_does_ubuntu_22041_get_released/) and [here](https://discourse.ubuntu.com/t/jammy-jellyfish-release-schedule/23906) suggest August 4th) or use the [link](https://osm2pgsql.org/doc/install.html) I provided earlier.
